releasenotes/notes/make-statevector-subscriptable-and-add-inner-product_method-a0337393d9a5b666.yaml

```diff
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    Make :class:`qiskit.quantum_info.states.statevector.Statevector` subscriptable.
+    User can now retrieve the i-th elementh in a Statevector by index as statevec[i].
+
+    See `#4916 <https://github.com/Qiskit/qiskit-terra/issues/4916>`__ from more information.
+
+    Add inner product method to :class:`qiskit.quantum_info.states.statevector.Statevector`.
+    It takes as imput an other statevector an outputs the inner product.
+
+    For example::
+      statevec_inner_other = statevec.inner(other)
+
+    See `#4934 <https://github.com/Qiskit/qiskit-terra/issues/4934>`__ from more information.
```

Comment:
For two separate features, it's best to put it in with two bullet points within the `features` key (so have two `- |` intro bits).